src/lib/factory.js

```diff
@@ -201,6 +208,21 @@ class PluginFactory extends EventEmitter2 {
     }
     return accounts
   }
+
+  * _handleGlobalNotification (account, notification) {
+    const eventParams = translateBellsToPluginApi(
+      notification,
+      account,
+      this.ledgerContext
+    )
+
+    // Inject the account as the first parameter
+    const eventType = eventParams[0]
+    const eventAdditionalParams = eventParams.slice(1)
+    const accountIlpAddress = this.ledgerContext.prefix + this.ledgerContext.accountUriToName(account)
+    const eventGlobalParams = [eventType].concat([accountIlpAddress], eventAdditionalParams)
```

Comment:
Couldn't this be `const eventGlobalParams = [eventType, accountIlpAddress].concat(eventAdditionalParams)`?